langchain/src/retrievers/time_weighted.ts

```diff
@@ -138,12 +153,15 @@ export class TimeWeightedVectorStoreRetriever extends BaseRetriever {
    * @returns An object containing salient documents and their scores
    */
   private async getSalientDocuments(
-    query: string
+    query: string,
+    callbacks?: Callbacks
   ): Promise<Record<number, { doc: Document; score: number }>> {
     const docAndScores: [Document, number][] =
       await this.vectorStore.similaritySearchWithScore(
         query,
-        this.searchKwargs
+        this.searchKwargs,
+        undefined,
+        callbacks
```

Comment:
Oh, I guess the above doesn't matter. Still maybe better to call `getChild()` here instead for clarity?